.github/workflows/reusable-build.yml

```diff
@@ -176,7 +176,7 @@ jobs:
     - name: Copy LLVM libs for Fuzzing & Address Sanitizing
       if: steps.skip_check.outputs.should_skip != 'true'
       working-directory: ./${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}}
-      run: copy "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.35.32215\bin\Hostx64\x64\clang*" .
+      run: copy "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\bin\Hostx64\x64\clang*" .
```

Comment:
can you use the VCToolsInstallDir environment variable to automatically get the right path and not have to change it when github updates the msvc version?

Reply:
@dthaler 
@saxena-anurag 

Yes, it can be done, but still the VS version and product type would have to be hard-coded (i.e. "2022", and "Enterprise"), if the current console does not have the VC environment variables:

```cmd
# Generate the Visual Studio environment variables (if not already done)
> call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"

# Parse the output string
> echo %VCToolsInstallDir%
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.36.32532\
```

Reply:
github preinstalls Visual Studio so that variable should always be set I think

Reply:
The VS environment variables are not installed as System defaults, i.e. that's why VS offers a "VS Command Prompt", which basically just runs that `.bat` script.

Reply:
I did a quick test and `%VCToolsInstallDir%` is not available by default in the runner.

Reply:
Maybe use the VSVARS32 script to get the variable set to save to github environment variables then for future steps.

Reply:
Update PR to use environment variable. Also updated the cache hash to use the VS tool version.